client/apps/landing/src/routes/claim.lazy.tsx

```
@@ -59,7 +58,7 @@ export const Route = createLazyFileRoute("/claim")({
 function Claim() {
   const {
     setup: {
-      systemCalls: { claim_leaderboard_rewards, register_to_leaderboard },
+      // systemCalls: { claim_leaderboard_rewards, register_to_leaderboard },
     },
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)